src/main/java/com/noxag/newnox/ui/configurationmodule/ConfigurationPane.java

```diff
@@ -1,28 +1,71 @@
-package com.noxag.newnox.ui.configurationmodule;
-
-import java.util.List;
-
-import javafx.scene.layout.Border;
-import javafx.scene.layout.BorderPane;
-import javafx.scene.layout.BorderStroke;
-import javafx.scene.layout.BorderStrokeStyle;
-import javafx.scene.layout.BorderWidths;
-import javafx.scene.layout.CornerRadii;
-import javafx.scene.paint.Color;
-
-public class ConfigurationPane extends BorderPane {
-    public ConfigurationPane() {
-        this.setCenter(new ConfigurationTabPane());
-
-        this.setBorder(new Border(
-                new BorderStroke(Color.BLACK, BorderStrokeStyle.SOLID, CornerRadii.EMPTY, BorderWidths.DEFAULT)));
-        // Add "run" button
-    }
-
-    public List<String> getTextanalyzer() {
-        // TODO: This method should return a list of strings containing every
-        // algorithm that has been "checked"
-        return null;
-    }
-
-}
+package com.noxag.newnox.ui.configurationmodule;
+
+import java.util.List;
+
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.control.Button;
+import javafx.scene.layout.Border;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.BorderStroke;
+import javafx.scene.layout.BorderStrokeStyle;
+import javafx.scene.layout.BorderWidths;
+import javafx.scene.layout.CornerRadii;
+import javafx.scene.layout.HBox;
+import javafx.scene.paint.Color;
+import javafx.stage.FileChooser;
+import javafx.stage.FileChooser.ExtensionFilter;
+
+public class ConfigurationPane extends BorderPane {
+
+    public Button btnrun, btnopen;
+    public FileChooser fileChooser;
+
+    public ConfigurationPane() {
+
+        ConfigurationTabPane configTabPane = new ConfigurationTabPane();
+        this.setCenter(configTabPane);
+
+        this.setBorder(new Border(
+                new BorderStroke(Color.BLACK, BorderStrokeStyle.SOLID, CornerRadii.EMPTY, BorderWidths.DEFAULT)));
+
+        // Add "run" button

```

Comment:
den kompletten code der den run button erstellt in eigene methode auslagern, also
private void createRunButton (){...}
und dann createRunButton() hier aufrufen